### DISCUSSION TITLE
Overflowing feeds on "latest change" container

### DISCUSSION THREAD

**Discussion by LaurelineP** [community/community#55994] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

## Problematic / Context
On github landing page the layout seems to have overflowing issue
- feeds box container overflow on the right container 

## Environment
Issue occured for me in those following circumstances:
- OS: mac
- browser: chrome
- github preferences: dark theme

### Unexpected behavior
Here is how the landing page once logged displays
We can see the feed box overflowing on the latest change timelines container 
<img width="1347" alt="image" src="https://github.com/community/community/assets/32878345/05997dce-eca8-4f74-9738-89bfb1a8a551">

### Insights
Here, something that may help in identifying where about could the issue be.
We can observe feed boxes having a width contained and not overflowing the latest changes container.
- the `.d-flex` classname was untogled through the navigator - which resulted as the following picture
<img width="1601" alt="image" src="https://github.com/community/community/assets/32878345/383198b5-eaae-40b1-bde5-6e40e05b3968">

**Comment by nkosinathintuli** [comment]
This has been going on for some time now here (https://github.com/orgs/community/discussions/55493)

Here's a comment we got from one of the maintainers 

> Hi there from GitHub 👋🏼 we are tracking this internally and investigating further. We appreciate your patience in the interim. I'll share an update here when I have one. Thank you all for your reporting, screenshots, and additional input.

Link to comment (https://github.com/orgs/community/discussions/55493#discussioncomment-5940498)

**Comment by cesarcruzr** [comment]
er
